hash_to_array.rb

```diff
@@ -0,0 +1,18 @@
+hash = {}
+key = nil
+5.times do
+  print 'Give me a key: '
+  key = gets.chomp
+  print 'Give me the value: '
+  hash[key] = gets.chomp
+end
+
+# I don't understand what's meant by the statement in
+# the instructions to "Write a function that accepts
+# the hash as 'optional' parameters." Is this what the
+# instructor want?
+def print_hash(hash = nil)
```

Comment:
Yep, I believe that is what the instructions mean. In this case you could do `hash = {}` to initialize it as an empty hash and then you wouldn't need the `unless hash.nil?`

Reply:
I did `def print_hash(hash = {})` then called `print_hash()`. The result was `[]` `[]`. That doesn't look good.